client/state/automated-transfer/selectors.js

```diff
@@ -48,3 +54,15 @@ export const getEligibility = compose(
 	getEligibilityData,
 	getAutomatedTransfer,
 );
+
+/**
+ * Checks if the site is currently transferring
+ *
+ * @param {Object} state global app state
+ * @param {number} siteId requested site for tranfer info
+ * @returns {bool} transferring check
+ */
+export const isAutomatedTransferTransferring = ( state, siteId ) => includes(
```

Comment:
I'm not sure I'm a fan of these long names. We already get the knowledge that this is about automated-transfer by means of the `import` statement where we might pull in `import { isTransferring } from 'state/automated-transfer/selectors'`